lib/vmdb/fast_gettext_helper.rb

```diff
@@ -1,23 +1,14 @@
 module Vmdb
   module FastGettextHelper
-    def self.human_locale(locale)
-      # TRANSLATORS: Provide locale name in native language (e.g. English, Deutsch or Português)
-      human_locale = _("locale_name")
-      human_locale = locale if human_locale == "locale_name"
-      human_locale
-    end
-
     def self.register_human_localenames
-      original_locale = FastGettext.locale
+      human_locale_names = YAML.load_file(Rails.root.join('config/human_locale_names.yaml'))['human_locale_names']
```

Comment:
Does this method need to be aware of supported_locales?  I'm not sure what this method is used for, so perhaps it doesn't matter, but if a locale is removed from the supported list, I guess I'd expect this list to not have it as well.